frontend/server/src/Controllers/User.php

```diff
@@ -3640,17 +3643,39 @@ public static function getProfileDetailsForSmarty(\OmegaUp\Request $r) {
                     'Identity'
                 );
             }
-            $smartyProperties = [
-                'profile' => self::getProfileDetails($r->identity, $identity),
-            ];
+            $profile =  self::getProfileDetails($r->identity, $identity);
+            $contests = self::apiContestStats($r);
+            $solvedProblems = self::apiProblemsSolved($r);
+            $unsolvedProblems = self::apiListUnsolvedProblems($r);
+            $createdProblems = self::apiProblemsCreated($r);
+            $stats = self::apiStats($r);
+            $badges = \OmegaUp\Controllers\Badge::apiList($r);
+            $ownedBadges = \OmegaUp\Controllers\Badge::apiMyList($r);
+            //print_r($stats);
+            //print_r($profile);
+            //print_r($unsolvedProblems);
+            //print_r($solvedProblems);
+            //print_r($createdProblems['problems']);
         } catch (\OmegaUp\Exceptions\ApiException $e) {
-            $smartyProperties = [
+            $profile = [
                 'STATUS_ERROR' => $e->getErrorMessage(),
             ];
         }
         return [
-            'smartyProperties' => $smartyProperties,
-            'template' => 'user.profile.tpl',
+            'smartyProperties' => [
+                'payload' => [
+                    'profile' => $profile,
+                    'contests' => $contests['contests'],
+                    'solvedProblems' => $solvedProblems['problems'],
+                    'unsolvedProblems' => $unsolvedProblems['problems'],
+                    'createdProblems' => $createdProblems['problems'],
+                    'stats' => $stats,
+                    'badges' => $badges,
+                    'ownedBadges' => $ownedBadges['badges']
+                ],
```

Comment:
Aquí como podrías obtener el error, puedes hacer lo siguiente para que almacenes toda la información del payload
```suggestion
                'payload' => array_merge($payload, [
                    'profile' => $profile,
                    'contests' => $contests['contests'],
                    'solvedProblems' => $solvedProblems['problems'],
                    'unsolvedProblems' => $unsolvedProblems['problems'],
                    'createdProblems' => $createdProblems['problems'],
                    'stats' => $stats,
                    'badges' => $badges,
                    'ownedBadges' => $ownedBadges['badges']
                ]),
```